tests/common/plugins/sanity_check/checks.py

```diff
@@ -982,3 +984,37 @@ def _check(*args, **kwargs):
         return init_check_result
 
     return _check
+
+
+# check ipv6 neighbor reachability
+@pytest.fixture(scope="module")
+def check_ipv6_mgmt(duthosts):
+    # check ipv6 mgmt interface reachability for debugging purpose only.
+    # No failure will be trigger for this sanity check.
+    def _check(*args, **kwargs):
+        init_result = {"failed": False, "check_item": "ipv6_mgmt"}
+        result = parallel_run(_check_ipv6_mgmt_to_dut, args, kwargs, duthosts, timeout=30, init_result=init_result)
+        return list(result.values())
+
+    def _check_ipv6_mgmt_to_dut(*args, **kwargs):
+        dut = kwargs['node']
+        results = kwargs['results']
+
+        logger.info("Checking ipv6 mgmt interface reachability on %s..." % dut.hostname)
+        check_result = {"failed": False, "check_item": "ipv6_mgmt", "host": dut.hostname}
+
+        # most of the testbed should reply within 10 ms, Set the timeout to 2 seconds to reduce the impact of delay.
+        try:
+            res = subprocess.check_output(["ping6", "-c", "2", str(dut.mgmt_ipv6)], stderr=subprocess.STDOUT)
```

Comment:
Use `localhost.shell("ping6 -c 2 " + dut.mgmt_ipv6)` may be easier than `subprocess`.

Reply:
Thanks @wangxin . Changed to use localhost.shell() now.